### PR TITLE
Open about section links externally in Settings

### PR DESCRIPTION
**Pull request by pierremtb**
Related to #4192

### CODE REVIEW

File: src/components/Settings/AllSettingsFields.tsx [hunk 1]
```diff
@@ -269,6 +270,9 @@ export const AllSettingsFields = forwardRef(
                 Want to experience the latest and (hopefully) greatest from our
                 main development branch?{' '}
                 <a
+                  onClick={openExternalBrowserIfDesktop(
+                    'https://zoo.dev/modeling-app/download/nightly'
+                  )}
```
Comment (pierremtb) [inline src/components/Settings/AllSettingsFields.tsx:275]: @franknoirot I've seen this done for the bottom right buttons so I'm replicating it here and it seems to work. Should this be done for the other about links above?

Comment (franknoirot) [reply src/components/Settings/AllSettingsFields.tsx:275]: Yeah I think it should, good catch!

Comment (pierremtb) [reply src/components/Settings/AllSettingsFields.tsx:275]: Ok will add!

Comment (pierremtb) [reply src/components/Settings/AllSettingsFields.tsx:281]: Done

**Pull request merged**